### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by raddevon**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report--client-library-.md [hunk 1]
```diff
@@ -20,10 +19,20 @@ Include the code that is causing the error:
...
+**Versions (please complete the following information):**
+
+<!--
+For EdgeDB Version: run `edgedb query 'select sys::get_version_as_str()'` from your project directory (or run `select sys::get_version_as_str();` in the EdgeDB interactive shell).
+For EdgeDB CLI version: Run `edgedb --version` from anywhere
+For `edgedb-js` version: Run `npm list edgedb --depth=0` from your project directory
+For Node/Deno version: Run `node --version` or `deno --version`
+-->
+
+- OS (e.g. iOS):
```
Comment (jaclarke) [inline .github/ISSUE_TEMPLATE/bug-report--client-library-.md:31]: I wasn't aware we supported iOS 😄.

File: .github/ISSUE_TEMPLATE/bug-report--query-builder-.md [hunk 2]
```diff
@@ -29,8 +29,17 @@ Run the `.toEdgeQL()` method on your query and print the result. Then copy the g
...
+For EdgeDB CLI version: Run `edgedb --version` from anywhere
+For `edgedb-js` version: Run `npm list edgedb --depth=0` from your project directory
+For Node/Deno version: Run `node --version` or `deno --version`
+-->
+
+- OS (e.g. iOS):
+- EdgeDB version (e.g. `2.0`):
+- EdgeDB CLI version (e.g. `2.0`):
+- `edgedb-js` version (e.g. `0.20.10`;):
+- Node/Deno version:
```
Comment (jaclarke) [inline .github/ISSUE_TEMPLATE/bug-report--query-builder-.md:45]: Can we also add `@edgedb/generate` version and typescript version.

Comment (raddevon) [reply .github/ISSUE_TEMPLATE/bug-report--query-builder-.md:49]: @jaclarke Added those two.

File: .github/ISSUE_TEMPLATE/bug-report--client-library-.md (hunk 1 above)
Comment (raddevon) [reply .github/ISSUE_TEMPLATE/bug-report--client-library-.md]: This has been removed

**Pull request merged**